FirebaseAppCheck.podspec

```diff
@@ -43,7 +43,7 @@ Pod::Spec.new do |s|
   s.tvos.weak_framework = 'DeviceCheck'
 
   s.dependency 'FirebaseCore', '~> 8.0'
-  s.dependency 'PromisesObjC', '>= 1.2', '< 3.0'
+  s.dependency 'PromisesObjC', '>= 2.0.0', '< 3.0'
```

Comment:
We should use the standard CocoaPods `~` syntax:

s.dependency 'PromisesObjC', '~> 2.0'

Reply:
Fixed